.github/workflows/test.yml

```diff
@@ -0,0 +1,68 @@
+name: test
+on:
+  pull_request:
+  push:
+    branches:
+      - develop
+      - 'releases/*'
+
+jobs:
```

Comment:
The e2e tests under here operate on the source in `dist/index.js`. The build step verifies that file is kept up to date with `src` so we can be confident the generated code we're testing is correct.